protocols/perf/src/bin/perf.rs

```diff
@@ -163,232 +163,48 @@ async fn client(
             .with(Protocol::Udp(server_address.port()))
             .with(Protocol::QuicV1),
     };
-
-    let benchmarks = if upload_bytes.is_some() {
-        vec![custom(
-            server_address,
-            RunParams {
-                to_send: upload_bytes.unwrap(),
-                to_receive: download_bytes.unwrap(),
-            },
-        )
-        .boxed()]
-    } else {
-        vec![
-            latency(server_address.clone()).boxed(),
-            throughput(server_address.clone()).boxed(),
-            requests_per_second(server_address.clone()).boxed(),
-            sequential_connections_per_second(server_address.clone()).boxed(),
-        ]
-    };
-
-    tokio::spawn(async move {
-        for benchmark in benchmarks {
-            benchmark.await?;
-        }
-
-        anyhow::Ok(())
-    })
-    .await??;
-
-    Ok(())
-}
-
-async fn custom(server_address: Multiaddr, params: RunParams) -> Result<()> {
```

Comment:
Removed all the other benchmark scenarios (e.g. parallel requests per minute) in favor of the one used in https://github.com/libp2p/test-plans/blob/master/perf/README.md. Despite what I said in the past, I don't think they are useful today. Easy to re-introduce in the future.